netbeans.apache.org/src/content/about/history.asciidoc

```diff
@@ -0,0 +1,166 @@
+contributed considerable amounts of code to open source projects over the
+years, this was Sun's first sponsored open source project, one in which Sun
+would be paying for the site and handling the infrastructure. The very first
+decision made was that it sounded logical to call the new site: NetBeans.org.
+In June 2000, the initial netbeans.org web site was launched. The years that
+followed focused on continual enhancements from release to release, as
+described in the section below.
+
+When Oracle acquired Sun in 2010, NetBeans became part of Oracle. Oracle
+actively seeked for new developers to work on the NetBeans team and sees
+NetBeans IDE as the official IDE for the Java Platform.
+
+In 2016 Oracle donated the source code of NetBeans to the link:https://www.apache.org/[Apache Software Foundation],
+and started as a podling in the link:http://incubator.apache.org/[Apache Incubator Project].
+
+During approximately two years and a half many volunteers joined the podling, and work started to
+adapt the codebase to comply with the Apache Software Foundation Guidelines. 
+The link:/community/index.html[Apache NetBeans Community] started growing, and in april 2019 the
+podling became a Top Level Apache Project.
+
+== Continual Enhancements from Release to Release
+
+An open source project is a living entity that needs time to find the right balance of people and contribution. Its growth is always an ongoing process.
+
+- The first year (through NetBeans 3.2), the project spent trying to find its
+  feet. The next few years involved learning about what worked in terms of
+open-source processes. (In the first two years, the development process was so
+open that more time was spent debating than implementing.) The growing pains
+paid off. With NetBeans 3.5, huge strides in performance were made, and tests
+and processes put in place to prevent regressions. With 3.6, the windowing
+system and property sheet were reimplemented, and the user interface cleaned up
+tremendously.
+
+- NetBeans 4 was a complete change in the way the IDE worked. A new project
+  system not only revamped the user experience, but also made it possible to
+replace infrastructure that had held the NetBeans back. The NetBeans 4.1
+release was built on the new project infrastructure of 4.0, and added more
+features and full J2EE support.  
+
+- NetBeans 5 introduced comprehensive support for developing IDE modules and
+  rich client applications based on the NetBeans platform; an intuitive GUI
+builder (Matisse); new and redesigned CVS support; support for Sun
+ApplicationServer 8.2, Weblogic 9 and JBoss 4.
+
+- NetBeans 6 focused on improved developer productivity through a rewritten,
+  smarter and faster editor, together with the integration of external NetBeans
+products into one IDE.
+
+- NetBeans 7 was released together with JDK 7, providing editor tools for
+  working with new JDK 7 language constructs, together with support for JavaFX
```

Comment:
Version 7.0 introduced support for OSGi: http://wiki.netbeans.org/Netigso
Version 7.1 started to reuse Eclipse components via OSGi.